src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -80,9 +80,24 @@ public class RepositoryCloneViewModel : BaseViewModel, IRepositoryCloneViewModel
                 loading => loading.Value || repositories.UnfilteredCount > 0 && !LoadingFailed)
                 .ToProperty(this, x => x.FilterTextIsEnabled);
 
-            this.WhenAny(x => x.IsLoading, x => x.LoadingFailed,
-                (loading, failed) => !loading.Value && !failed.Value && repositories.UnfilteredCount == 0)
-                .Subscribe(x => NoRepositoriesFound = x);
+            this.WhenAny(
+                x => x.repositories.UnfilteredCount,
```

Comment:
Ok, this is happening because in the unit tests, `repositories.OriginalCompleted` is null.

Previously the unit tests were replacing the tracking collection with another tracking collection in the now removed assignment: one that had a source observable and therefore an OriginalCompleted.